apps/emqx_resource/src/emqx_resource_manager.erl

```diff
@@ -702,3 +702,9 @@ safe_call(ResId, Message, Timeout) ->
         exit:{timeout, _} ->
             {error, timeout}
     end.
+
+%% the config and state of a bridge often contains some sensitive data
+%% we shouldn't expose them to logs
+redact_data(Data) ->
+    Msg = <<"this data is redacted due to security reasons">>,
```

Comment:
this is as good as not showing anything at all?
why doesn't `emqx_utils:redact` work here?